lib/file_download.rb

```diff
@@ -1,8 +1,12 @@
 require "net/http"
 
-# Used to download a remote file. Handles redirects.
+# Used to download a remote file. Handles redirects up to a maximum of 5.
```

Comment:
Is it worth mentioning that this can be configured?

Reply:
It can't be configured. The value of 5 comes from the `FileDownload::MAX_REDIRECTS` constant and the place it is used is a private method.

Reply:
👍